### PR TITLE
Aavruti_Coding_Exam_ValidatePIN

### PR DESCRIPTION
**Pull request by AavrutiDJun20**
image

### CODE REVIEW

File: src/aavrutiDJun20/ValidatePinNumber.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+					num += number.charAt(index);
+				}
+			}
+			if(flag) {
+				char charArray[] = number.toCharArray();
+				Arrays.sort(charArray);
+				String numSorted = String.valueOf(charArray);
+				
+				StringBuilder strReverse = new StringBuilder(numSorted);
+				strReverse.reverse();
```
Comment (harshvegada) [inline src/aavrutiDJun20/ValidatePinNumber.java:29]: Hi @AavrutiDJun20 
I feel using this reverse() method we not achieving our goal, Here we need to check ascending and descending order, not reverse order. check once

**Review by harshvegada — APPROVED** [review]
very nice @AavrutiDJun20 
As expected

File: src/aavrutiDJun20/ValidatePinNumber.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	
+	void validatePin(String number) {
+		String num = "";
+		boolean flag = true;
+		
+		try {
+			Integer.parseInt(number);
+			
+			for(int index=0;index<number.length();index++) {
+				if(num.contains(String.valueOf(number.charAt(index)))) {
```
Comment (KrishnaKTechnocredits) [inline src/aavrutiDJun20/ValidatePinNumber.java:15]: Instead of writing loop you can use try catch --> NumberFormateException

Comment (KrishnaKTechnocredits) [reply src/aavrutiDJun20/ValidatePinNumber.java:15]: try{
flag= true
}catch(NumberFormateException ne){
flag = false
}

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**